src/test/java/spoon/test/visitor/AssignmentsEqualsTest.java

```diff
@@ -20,12 +23,11 @@ public void testEquals() {
 
 		Factory factory = launcher.getFactory();
 		List<CtAssignment> assignments = Query.getElements(factory, new TypeFilter<>(CtAssignment.class));
-
-		assertTrue(assignments.size() == 10);
-		assertFalse(assignments.get(0).equals(assignments.get(1)));
-		assertFalse(assignments.get(2).equals(assignments.get(3)));
-		assertFalse(assignments.get(4).equals(assignments.get(5)));
-		assertFalse(assignments.get(6).equals(assignments.get(7)));
-		assertTrue(assignments.get(8).equals(assignments.get(9)));
+		assertSame(assignments.size(), 10);
```

Comment:
should be assertEquals for primitive types

Reply:
I fixed it, but I think we should first migrate to JUnit 5 and then search and fix the style bugs. Otherwise, the PRs are pretty big and complex to review.